v2/internal/typescriptify/typescriptify.go

```diff
@@ -901,3 +907,20 @@ func differentNamespaces(namespace string, typeOf reflect.Type) bool {
 	}
 	return false
 }
+
+func typeClashWithReservedKeyword(input string) bool {
+	in := strings.ToLower(strings.TrimSpace(input))
+	for _, v := range jsReservedKeywords {
+		if in == v {
+			return true
+		}
+	}
+
+	return false
+}
+
+func warnAboutTypesClash(entity string) {
+	l := log.New(os.Stderr, "", 0)
```

Comment:
```suggestion
	// TODO: Refactor logging
	l := log.New(os.Stderr, "", 0)
```
Adding this as a reminder for future self that we need to refactor logging, support better warnings, honour silent flag